apps/src/code-studio/browserNavigation.js

```diff
@@ -7,10 +7,14 @@ import {getStore} from '../redux';
 import {setCurrentLevelId} from '@cdo/apps/code-studio/progressRedux';
 
 // Returns whether we can safely navigate between the two given level apps
-// without reloading the whole page.  For now, this only works when moving
-// from a "music" level to another "music" level.
+// without reloading the whole page.
 export function canChangeLevelInPage(currentLevelApp, newLevelApp) {
-  return currentLevelApp === 'music' && newLevelApp === 'music';
+  const compatibleApps = ['music', 'standalone_video2'];
```

Comment:
this logic feels like it is opening us up to allowing switches between any app type. Do we want that? It might be fine, it just feels like a departure from our previous thoughts.

Reply:
I expect a more likely situation is that we "family up" the sets that we can switch between, and had always kept that in the back of my mind.  We still restrict these switches to those within the same lesson, so I think it makes sense that we can switch between more than one app type, but only a known subset within that given lesson.